components/fxa-client/src/http_client.rs

```diff
@@ -344,6 +357,54 @@ impl FxAClient for Client {
         Ok(())
     }
 
+    fn attached_clients(
+        &self,
+        config: &Config,
+        refresh_token: &str,
+    ) -> Result<ResponseAndETag<Vec<GetAttachedClientResponse>>> {
+        let url = config.auth_url_path("v1/account/attached_clients")?;
+        let request =
+            Request::get(url).header(header_names::AUTHORIZATION, bearer_token(refresh_token))?;
+        let resp = Self::make_request(request)?;
+        let attached_clients: Vec<GetAttachedClientResponse> = resp.json()?;
+        let etag = resp
+            .headers
+            .get(header_names::ETAG)
+            .map(ToString::to_string);
+        Ok(ResponseAndETag {
+            etag,
+            response: attached_clients,
+        })
+    }
+
+    fn destroy_attached_client(
+        &self,
+        config: &Config,
+        refresh_token: &str,
+        client_id: &str,
+        session_token_id: Option<String>,
+        device_id: Option<String>,
+    ) -> Result<()> {
```

Comment:
According to [the docs](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/docs/api.md#post-accountattached_clientdestroy), `refresh_token`, `client_id`, `session_token_id`, and `device_id` are optional.  I've made a call here to balance functionality and safety but it may not be the right one. Let me know if that's the case.

Reply:
@rfk might have an opinion here as I don't?

Reply:
I'm going to add him as a reviewer since I'm implementing the debouncing logic he recommended for the get devices functionality.

Reply:
Thanks for this feedback. I'm okay with removing the destroy functionality. @vladikoff does that work for you? I think we discussed adding that functionality to this ticket.

Reply:
@lougeniaC64 Yes that's fine, I double checked and we won't need it in there . Let's remove the destroy functionality